Public/app/Components/NavBar.jsx

```diff
@@ -11,9 +11,7 @@ var NavBar = ({logout, savePrefs, synced}) => (
           Try interacting with the ticker graphs on the right! In your account, you can customize your 
           dashboard view any way you like, and save those preferences so that you see them each time you log in. 
         </span>
-        <form action="/login" method="get">
-          <button className="btn btn-default"></button>
-        </form>
+        <button className="btn btn-default" onClick={() => window.location ='/login'}>Logout</button>
```

Comment:
And took away the form wrapping. Clarabelle and I were just conjuring with that form element